backend/modules/professor/routers/professor_cohorts.py

```
@@ -70,7 +70,7 @@ async def refresh_assigned_simulations(
         return {"status": "ok", **result}
     except Exception as e:
         logger.error(f"Error in refresh_assigned_simulations: {str(e)}", exc_info=True)
```

Ruff (0.15.7)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)